test_average (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... FAIL
test_to_degrees (__main__.TestExercice) ... ERROR
test_to_farenheit (__main__.TestExercice) ... FAIL
test_to_radians (__main__.TestExercice) ... ok

======================================================================
ERROR: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 56, in test_to_degrees
    output = [exercice.to_degrees(v) for v in values]
  File "test_exercice.py", line 56, in <listcomp>
    output = [exercice.to_degrees(v) for v in values]
  File "/home/runner/work/c01_ch3_exercices-domlessard/c01_ch3_exercices-domlessard/exercice.py", line 22, in to_degrees
    return (deg, minutes, secondes)
NameError: name 'minutes' is not defined

======================================================================
FAIL: test_to_celsius (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 74, in test_to_celsius
    'Mauvais calcul de farenheit -> celsius'
AssertionError: Lists differ: [0.0, 0.0, 0.0] != [-17.77777777777778, -31.666666666666664, 232.77777777777777]

First differing element 0:
0.0
-17.77777777777778

- [0.0, 0.0, 0.0]
+ [-17.77777777777778, -31.666666666666664, 232.77777777777777] : Mauvais calcul de farenheit -> celsius

======================================================================
FAIL: test_to_farenheit (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 86, in test_to_farenheit
    'Mauvais calcul de celsius -> farenheit'
AssertionError: Lists differ: [0.0, 0.0, 0.0] != [32.0, -13.0, 843.8000000000001]

First differing element 0:
0.0
32.0

- [0.0, 0.0, 0.0]
+ [32.0, -13.0, 843.8000000000001] : Mauvais calcul de celsius -> farenheit

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (failures=2, errors=1)
